build.gradle

```diff
@@ -60,9 +60,9 @@ dependencies {
 	modLocalRuntime "dev.emi:emi-fabric:${project.emi_version}"
 
 	// Renderer (https://github.com/0x3C50/Renderer)
-	include modImplementation("com.github.0x3C50:Renderer:${project.renderer_version}") {
-		exclude group: "io.github.ladysnake" exclude module: "satin"
-	}
+	//include modImplementation("com.github.0x3C50:Renderer:${project.renderer_version}") {
+		//exclude group: "io.github.ladysnake" exclude module: "satin"
+	//}
```

Comment:
Just delete this section? Also remember to delete version in `gradle.properties`